network/ClientNetworking.h

```diff
@@ -67,6 +67,9 @@ class ClientNetworking : public std::enable_shared_from_this<ClientNetworking> {
 
     /** Checks if the client has some authorization \a role. */
     bool HasAuthRole(Networking::RoleType role) const;
+
+    /** Returns destination address of server. */
```

Comment:
not clear what a "destination address" is. could you elaborate?

Reply:
It's a address of multiplayer server entered by player in `ServerConnectWnd`.

Reply:
elaborate in the comment